buildspec.yml

```diff
@@ -0,0 +1,63 @@
+version: 0.2
+
+env:
+  variables:
+    FRAMEWORK_FULL_VERSION: '1.4.1'
+    FRAMEWORK_SHORT_VERSION: '1.4'
+    AWS_DEFAULT_REGION: 'us-west-2'
+    ECR_REPO: 'sagemaker-mxnet-serving'
+    GITHUB_REPO: 'sagemaker-mxnet-serving-container'
+    EI_ACCELERATOR_TYPE: 'ml.eia1.medium'
+    GPU_INSTANCE_TYPE: 'p2.xlarge'
+    SETUP_FILE: 'setup_cmds.sh'
+    SETUP_CMDS: '#!/bin/bash\npip install --upgrade pip\npip install -U -e .\npip install -U -e .[test]'
+
+phases:
+  pre_build:
+    commands:
+      - start-dockerd
+
+      - ACCOUNT=$(aws sts get-caller-identity --query 'Account' --output text)
+
+  build:
+    commands:
+      # run linter
+      - tox -e flake8
+
+      # run unit tests
+      - tox -e py27,py36 test/unit
+
+      # create pip archive (tar name will be something like sagemaker_mxnet_serving_container-1.0.0.tar.gz, but the Dockerfiles expect sagemaker_mxnet_serving_container.tar.gz)
+      - python3 setup.py sdist
+      - mv dist/sagemaker_mxnet_serving_container-*.tar.gz dist/sagemaker_mxnet_serving_container.tar.gz
+      - cp dist/sagemaker_mxnet_serving_container.tar.gz docker/$FRAMEWORK_FULL_VERSION/py3/sagemaker_mxnet_serving_container.tar.gz
+
+      # build images
+      - python3 scripts/build_all.py --version $FRAMEWORK_FULL_VERSION --account $ACCOUNT --repo $ECR_REPO
+
+      # run cpu local integration tests
+      - IGNORE_COVERAGE=- tox -e py36 -- test/integration/local --py-version 2,3 --processor cpu --framework-version $FRAMEWORK_FULL_VERSION --region $AWS_DEFAULT_REGION --docker-base-name $ECR_REPO
+
+      # push docker images to ECR
+      - python3 scripts/publish_all.py --version $FRAMEWORK_FULL_VERSION --account $ACCOUNT --repo $ECR_REPO
+
+      # launch remote gpu instance
+      - create-key-pair
+      - launch-ec2-instance --instance-type $GPU_INSTANCE_TYPE --ami-name dlami-ubuntu
+
+      # run gpu local integration tests
+      - printf "$SETUP_CMDS" > $SETUP_FILE
+      - ecr_image="$ACCOUNT.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com/$ECR_REPO"
+      - cmd="IGNORE_COVERAGE=- tox -e py36 -- test/integration/local --processor gpu --py-version 2,3 --framework-version $FRAMEWORK_FULL_VERSION --region $AWS_DEFAULT_REGION --docker-base-name $ecr_image"
+      - remote-test --test-cmd "$cmd" --github-repo $GITHUB_REPO --branch master --setup-file $SETUP_FILE
+
+      # run sagemaker integration tests
+      - IGNORE_COVERAGE=- tox -e py36 -- test/integration/sagemaker -n 16 --py-version 2,3 --processor cpu,gpu --region $AWS_DEFAULT_REGION --docker-base-name $ECR_REPO --aws-id $ACCOUNT --framework-version $FRAMEWORK_FULL_VERSION --reruns 3 --reruns-delay 5
+
+      # run ei tests
+      - IGNORE_COVERAGE=- tox -e py36 -- test/integration/sagemaker/test_elastic_inference.py -n 2 --py-version 2,3 --processor cpu --accelerator-type $EI_ACCELERATOR_TYPE --region $AWS_DEFAULT_REGION --docker-base-name "$ECR_REPO-eia" --aws-id $ACCOUNT --framework-version $FRAMEWORK_FULL_VERSION --reruns 3
+
+    finally:
```

Comment:
also might want to delete the images: https://github.com/aws/sagemaker-mxnet-container/blob/master/buildspec.yml#L110

Reply:
Cleaned up test images in the end